plugins/module_utils/network/junos/facts/legacy/base.py

```diff
@@ -162,6 +164,10 @@ def populate(self):
             for child in obj:
                 if child.text != "\n":
                     mod.update({child.tag.replace("-", "_"): child.text})
+                if "chassis-sub-module" in child.tag:
+                    mod["chassis_sub_module"] = xmltodict.parse(tostring(obj))["chassis-module"][
```

Comment:
```suggestion
                    mod["chassis_sub_module"] = self._get_xml_dict(obj)["chassis-module"][
                        "chassis-sub-module"
                    ]
```

Reply:
@elchico2007  we can have wrapper similar to https://github.com/ansible-collections/junipernetworks.junos/blob/bd2bad901b7557288c7eec3607925e44009d2dec/plugins/module_utils/network/junos/facts/bgp_global/bgp_global.py#L142

so that we failed with required lib missing message when xmltodict not installed.